lib/python/qmk/cli/generate/api.py

```diff
@@ -0,0 +1,58 @@
+"""This script automates the generation of the QMK API data.
+"""
+from pathlib import Path
+from shutil import copyfile
+from time import strftime
+import json
+
+from milc import cli
+
+from qmk.info import info_json
+from qmk.keyboard import list_keyboards
+
+
+@cli.subcommand('Creates a new keymap for the keyboard of your choosing', hidden=False if cli.config.user.developer else True)
+def generate_api(cli):
+    """Generates the QMK API data.
+    """
+    api_data_dir = Path('api_data')
+    v1_dir = api_data_dir / 'v1'
+    keyboard_list = v1_dir / 'keyboard_list.json'
+    keyboard_all = v1_dir / 'keyboards.json'
+    usb_file = v1_dir / 'usb.json'
+
+    if not api_data_dir.exists():
+        api_data_dir.mkdir()
+
+    kb_all = {'last_updated': strftime('%Y-%m-%d %H:%M:%S %Z'), 'keyboards': {}}
+    usb_list = {'last_updated': strftime('%Y-%m-%d %H:%M:%S %Z'), 'devices': {}}
```

Comment:
```suggestion
    time = datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S UTC')
    kb_all = {'last_updated': time, 'keyboards': {}}
    usb_list = {'last_updated': time, 'devices': {}}
```
I think keeping the time in UTC would make more sense.

Reply:
Yeah, UTC is a good idea. I've gone a little farther than your suggestion in fe66b940e though. :)